R/modify_legend.R

```diff
@@ -208,3 +214,32 @@ modify_legend <- function(object, location="topright", legend_offset=0.3, draw=F
   object[['legend']][[paste0("legend.", legend.index)]] <- legend.config
   return(object)
 }
+
+#' get vector of legend arguments - overall or for each entry
+#' 
+#' @param overall logical indicating whether overall legend arguments should be returned. These arguments impact 
+#' the legend style (e.g. title, box.lwd, location, etc)
+#' @param indiv logical indiciating whether arguments applicable to each legend entry should be returned. These are 
+#' the arguments that apply to each individual entry within the legend (e.g. lty, lwd, pch, text.font, etc)
+#' @keywords internal
+get_legend_arg_names <- function(overall = FALSE, indiv = FALSE, names.args = names(formals(graphics::legend))){
```

Comment:
well I don't think it needs to be an argument now that I am looking again. The idea was that `names.args` is just always the formals of `legend`, so that isn't a good use of a function argument. It should really just go inside the function